frontend/packages/topology/src/components/page/TopologyPageToolbar.tsx

```diff
@@ -56,6 +56,7 @@ const TopologyPageToolbar: React.FC<TopologyPageToolbarProps> = observer(
             onClick={() =>
               onViewChange(showGraphView ? TopologyViewType.list : TopologyViewType.graph)
             }
+            aria-label="topology_view_switcher"
```

Comment:
`aria-label` should be translated. Why does it have underscores?

Reply:
fixed